### PR TITLE
#164829349 Upgrade Django version to 2.1

### PR DESCRIPTION
**Pull request by LarryKarani**
__What does PR do?__
Upgrades Authors Haven Django app from Django 1.10 to Django 2.1

__Description of tasks to be completed__

- Check for deprecation warnings.
- Upgrade Django from version 1.10 to version 2.1.7.
- Update requirements.txt with the upgraded Django version.

__Relevant Pivotal tracker stories__
https://www.pivotaltracker.com/story/show/164829349

### CODE REVIEW

**Review by MainaKamau92 — CHANGES_REQUESTED** [review]
Consider using Pipfile as opposed to requirements.txt for both uniformity and heroku's preference for pipfile over requirements

**Review by Kyppy — DISMISSED** [review]
Nice and concise commit.

**Review by MainaKamau92 — DISMISSED** [review]
Well done commit

**Review by MainaKamau92 — APPROVED** [review]

**Review by MainaKamau92 — DISMISSED** [review]

File: authors/settings.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
     'django.contrib.staticfiles',
 
     'corsheaders',
-    'django_extensions',
+
```
Comment (WNjihia) [inline authors/settings.py]: Hey @larryTheGeek, I'm curious about this. Why did you choose to remove django extensions?

**Review by WNjihia — CHANGES_REQUESTED** [review]
I've also noticed you've included both the pipfile and the requirements.txt. This seems redundant as they both do the same thing. Could you kindly decide (together with your team members) on which of the two to use and make the necessary changes

**Comment by WNjihia** [comment]
Seems like we have some merge conflicts. Kindly fix them so I'll be able to merge

File: authors/settings.py (hunk 1 above)
Comment (LarryKarani) [reply authors/settings.py]: It was a mistake. Let me rectify it immediately

**Review by MainaKamau92 — DISMISSED** [review]

**Review by WNjihia — DISMISSED** [review]

**Comment by WNjihia** [comment]
@larryTheGeek seems like we have a merge conflict. Kindly fix it so I can be able to merge

**Review by MainaKamau92 — APPROVED** [review]

**Review by WNjihia — APPROVED** [review]

**Pull request merged**